external/buildscripts/sources.txt

```diff
@@ -1,535 +1,541 @@
-####################

```

Comment:
Whitespace?

Reply:
This file should be LF, according to the .editorconfig. It looks like it was changed to CRLF at some point.